rpi/Animal/lib/Animal.pm

```diff
@@ -29,107 +31,103 @@ Perhaps a little code snippet.
     my $foo = Animal->new();
     ...
 
-=head1 EXPORT
-
-A list of functions that can be exported.  You can delete this section
-if you don't export anything, such as for a purely object-oriented module.
-
 =head1 SUBROUTINES/METHODS
 
-=head2 function1
+=head2 named
 
-=cut
+Create an animal with a name
 
-sub speaks_english {
-  return 0;
-}
+=cut
 
-=head1 AUTHOR
```

Comment:
You threw away lots of perldoc. Guess it makes the file easier to read, but the diff harder.